launchable/test_runners/nunit.py

```diff
@@ -15,9 +15,13 @@ def build_path(e: Element):
         pp = e.parent.tags.get('path') or []    # type: ignore
     if e.name == "test-suite":
         # <test-suite>s form a nested tree structure so capture those in path
-        e.tags['path'] = pp + [{'type': e.attrs['type'], 'name': e.attrs['name']}]
+        pp = pp + [{'type': e.attrs['type'], 'name': e.attrs['name']}]
     if e.name == "test-case":
-        e.tags['path'] = pp + [{'type': 'TestCase', 'name': e.attrs['name']}]
+        pp = pp + [{'type': 'TestCase', 'name': e.attrs['name']}]
+
+    if len(pp) > 0:
+        # remove file path prefix
+        e.tags['path'] = [{**path, 'name': path['name'].split(os.sep)[-1]} for path in pp]
```

Comment:
This is too broad for my comfort. IThis processing kicks in for every name, not just assembly name. I'm specifically worried about parameterized tests where names seem to include parameter values. I'd imagine it'd be possible for that to contain a path separator.

When we restrict this to just assembly names, it'd be also prudent to split not just by `os.sep` but either `\` or `/` -- in a complicated CI setup people often transfer test reports from one location to another, for example when parallelization is involved, so it's conceivable that some tests might run on Windows and then collected in Linux, and vice versa. Accepting both gracefully therefore would be a little more forgiving.

Reply:
Okay, it's a rough way. I'm going to fix this to focus on `Assembly` path, and to support both `/` and `\` regardless of OS.